components/rhs_thread/rhs_thread.tsx

```diff
@@ -97,6 +98,8 @@ export default class RhsThread extends React.Component<Props, State> {
             topRhsPostId: '',
             openTime,
         };
+
+        this.rhspostlist = React.createRef();
```

Comment:
Can we rename this to `this.rhspostlistRef`, to be consistent with other instances and PR's

Reply:
Sure